src/components/text/Link.jsx

```diff
@@ -18,7 +18,8 @@ const SIZE = {
   OBSCURE: 'obscure'
 };
 
-const Link = ({children, href = '#', color, size = SIZE.NORMAL, unstyled, underlined, emphasised, disabled}) => {
+const Link = ({children, href = '#', color, size = SIZE.NORMAL, unstyled, underlined, emphasised, disabled,
```

Comment:
@kdzwinel, @matzimowski, @burakukula, @matikrk, @vergilius what do you think about it?